azure-iot-device/azure/iot/device/constant.py

```diff
@@ -0,0 +1,12 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module defines constants for use across the azure-iot-device package
+"""
+
+VERSION = "2.0.0-preview.8"
+USER_AGENT = "py-azure-iot-device/{version}".format(version=VERSION)
```

Comment:
>py- [](start = 14, length = 3)

is this just constructed by us ? or is there a format that we are supposed to follow ? asking because I did not see `py` before anywhere, but may be mistaken. And if there is no format then why don't we use `python` fully ?

Reply:
This is whatever we want it to be. It's entirely arbitrary. I chose 'py' to make it shorter, and since 'py' is the accepted abbreviation for Python (pypi, pytest, pycharm, the py library itself, etc.)

---
In reply to: [307825444](https://github.com/Azure/azure-iot-sdk-python-preview/pull/147#discussion_r307825444) [](ancestors = 307825444)